VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -50,6 +50,12 @@ public static void setAuthorAttributionText(@NonNull TextView textView,
             final CharSequence formattedTime = getRelativeTimeSpanString(context,
                     initialTimeStampMs, authorData.getCreatedAt().getTime());
             final String authorLabel = authorData.getAuthorLabel();
+            final StringBuilder authorLabelFormatted = new StringBuilder();
+            if (authorLabel != null) {
+                authorLabelFormatted.append("(");
+                authorLabelFormatted.append(authorLabel);
+                authorLabelFormatted.append(")");
```

Comment:
It's always better to localize as much as possible of the string content. Other languages might have a different type of parentheses, or they might use a different method to express them (which you can't know unless you're familiar with all the languages in the world :stuck_out_tongue:). Basically, there doesn't seem to be any reason _not_ to have it in the strings themselves, and it will simplify the code as well.